src/workloads/execution/VariadicAggregateExpressions.yml

```diff
@@ -0,0 +1,621 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-execution"
+Description: |
+  This workload measures the performance of aggregation expressions that
+  can take in a variable number of expressions as arguments.
+
+  While measuring performances, this workload collects numbers
+  for either the SBE or the classic engine aggregation expression
+  implementations, depending on environments that it runs on.
+
+  Numbers on the 'standalone-all-feature-flags' environment are for
+  the SBE variadic aggregation expressions and numbers on the 'standalone'
+  environment for the classic variadic aggregation expressions.
+
+Keywords:
+- aggregate
+- sbe
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  DocumentCount: &DocumentCount 1e6
+  Repeat: &Repeat 10
+  Threads: &Threads 1
+  MaxPhases: &MaxPhases 32
+  TenIntFields: &TenIntFields ["$int1", "$int2", "$int3", "$int4", "$int5",
```

Comment:
can we use the new deterministic ^Choose to auto generate these? As in, something like ^Join together 'fieldname' and a choose array from 1 to 10? 

Similar comment for the document schema, but if you're going to do that, it might be worth leaving a comment describing the schema.

Reply:
Done.